packages/x-date-pickers/src/DateTimeField/tests/timezone.DateTimeField.test.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { DateTimeField } from '@mui/x-date-pickers/DateTimeField';
+import { screen, act, userEvent, fireEvent } from '@mui/monorepo/test/utils';
+import { createPickerRenderer, expectInputValue } from 'test/utils/pickers-utils';
+
+describe('<DateTimeField /> - TimeZone', () => {
+  describe('Value time-zone modification - Luxon', () => {
+    const { render, adapter } = createPickerRenderer({ clock: 'fake', adapterName: 'luxon' });
+    it('should update the field when dayjs', () => {
```

Comment:
What does `dayjs` mean in here?
It seems that we are testing `luxon` adapter... Maybe it should say something else?
```suggestion
    it('should update the field when time zone changes (timestamp remains the same)', () => {
```

Reply:
Effectively, I wanted to do dayjs, but I changed for Luxon since it's the one used in the issue